flink-table/flink-table-common/src/test/java/org/apache/flink/table/catalog/CatalogTestUtil.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.catalog;
+
+import org.apache.flink.api.common.typeinfo.BasicTypeInfo;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.api.java.typeutils.RowTypeInfo;
+import org.apache.flink.table.api.TableSchema;
+import org.apache.flink.table.plan.stats.TableStats;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Utility class for catalog testing.
+ */
+public class CatalogTestUtil {
+
+	public static GenericCatalogTable createTable() {
+		TableSchema tableSchema = TableSchema.fromTypeInfo(getRowTypeInfo());
+		return createTable(tableSchema, createTableStats(), new HashMap<String, String>());
+	}
+
+	public static RowTypeInfo getRowTypeInfo() {
+		return new RowTypeInfo(
+			new TypeInformation[] {BasicTypeInfo.INT_TYPE_INFO, BasicTypeInfo.INT_TYPE_INFO},
+			new String[] {"a", "b"});
+	}
+
+	public static TableStats createTableStats() {
+		return new TableStats(2);
+	}
+
+	public static GenericCatalogTable createTable(TableSchema schema, Map<String, String> tableProperties) {
+		return createTable(schema, new TableStats(0), tableProperties);
+	}
+
+	public static GenericCatalogTable createTable(TableSchema schema, TableStats stats,
+		Map<String, String> tableProperties) {
+		return new GenericCatalogTable(schema, stats, tableProperties);
+	}
+
+	public static void compare(GenericCatalogTable t1, GenericCatalogTable t2) {
```

Comment:
Maybe change `compare` to `assertEquals` is more accurate? due to `compare` in general will return an INT.  What do you think?